langchain/prompts/prompt.py

```diff
@@ -43,6 +43,42 @@ def lc_attributes(self) -> Dict[str, Any]:
     validate_template: bool = True
     """Whether or not to try validating the template."""
 
+    def __add__(self, other: Any) -> PromptTemplate:
+        # Allow for easy combining
+        if isinstance(other, PromptTemplate):
+            if self.template_format != "f-string":
+                raise ValueError(
+                    "Adding prompt templates only supported for f-strings."
+                )
+            if other.template_format != "f-string":
+                raise ValueError(
+                    "Adding prompt templates only supported for f-strings."
+                )
+            input_variables = list(
+                set(self.input_variables) | set(other.input_variables)
+            )
+            template = self.template + "\n\n" + other.template
```

Comment:
It feels a bit restricting to always insert the double newline, especially if we can do concatenation with strings directly. I think I'd rather be able to do 

```
prompt = (
    PromptTemplate.from_template("Tell me a joke about {topic}")
    + "\n\nmake it funny"
    + ", and in {language}"
)
```

And be explicit rather than have that be magical.